support-lambdas/stripe-intent/build.sbt

```diff
@@ -13,6 +13,7 @@ riffRaffManifestBranch := Option(System.getenv("BRANCH_NAME")).getOrElse("unknow
 riffRaffBuildIdentifier := Option(System.getenv("BUILD_NUMBER")).getOrElse("DEV")
 assemblyMergeStrategy in assembly := {
   case x if x.endsWith("module-info.class") => MergeStrategy.discard
+  case str if str.contains("simulacrum") => MergeStrategy.first
```

Comment:
there is a simulacrum-scalafix in cats, and the normal simulacrum also.  This causes a conflict due to version differences at the assembly staage, for both this and support-workers jar.